### ISSUE TITLE
Please add DAISY 3 support

### ISSUE THREAD

**Issue by rpoitevin** [Skywalker13/DaisyDuck#4]
Dear taem,

I'm very interested in this software. However, what a pity to support only DAISY 2 and no DAISY 3, when daisy-player already supports it. For instance, francophone libraries build last audio books only in DAISY 3, so it's impossible to access to new books.

Thanks for this project!

Regards,

Raphaël

**Comment by Skywalker13** [comment]
Hello,
It depends if I can found a C/C++ library for handling Daisy 3. I written the library for Daisy 2.02 but Daisy 3, IIRC, has a lot of major changes.

**Comment by alexarnaud** [comment]
Hello @Skywalker13 ,

The daisy-player developer provides a library for parsing Daisy 3 in C: http://jlemmens.nl/

Best regards.

**Comment by Skywalker13** [comment]
Yes it needs a major update, like for example migrate to Qt5 by the way. It should be the first thing to do. I must rework the whole building process because it's a bit old and I've much better scripts/tools in other unrelated project that I can use here.

The main problem is my free time.

**Comment by ZeroDot1** [comment]
I am very interested in using utilities like your software and helping to translate it.
Right now your program is the only one with a working gui for Linux that works, and it would be very helpful for many to have it updated.

That there are problems with the free time is clear for me, but I am sure that it is worth to develop the software further.
I am blind myself, and would be very happy if I could support you with the German translation.

Translated with www.DeepL.com/Translator
